cli/flags.rs

```diff
@@ -4371,6 +4522,31 @@ mod tests {
     );
   }
 
+  #[test]
+  fn test_watch_with_no_clear_screen() {
```

Comment:
Looks good 👍 please add a test case that uses `--no-clear-screen` but doesn't use `--watch` and assert that it returns error